amgut/handlers/auth_handlers.py

```diff
@@ -38,6 +39,12 @@ def post(self):
         printresults = ag_data.checkPrintResults(skid)
         if printresults is None:
             printresults = 'n'
+
+        # make sure kit has bcrypt encrypted password, and fix if not
+        if not bcrypt.identify(kitinfo['password']):
```

Comment:
`bcrypt.identify` returns False if the argument is not a bcrypt hash; the docs are [here](http://pythonhosted.org//passlib/password_hash_api.html?highlight=identify#passlib.ifc.PasswordHash.identify) (it's a member of all of the different hash modules so they just documented it in one place)